modules/ROOT/pages/clauses/where.adoc

```diff
@@ -555,49 +560,21 @@ Properties can also be added to patterns:
 .Query
 [source, cypher]
 ----
-MATCH (n:Person)
-WHERE (n)-[:KNOWS]-({name: 'Timothy'})
-RETURN n.name, n.age
+MATCH (other:Person)
+WHERE (other)-[:KNOWS]-({name: 'Timothy'})
+RETURN other.name, other.age
 ----
 
 The `name` and `age` values are returned for nodes that have a relationship with the type `KNOWS` connected to `Timothy`:
 
 .Result
 [role="queryresult",options="header,footer",cols="2*<m"]
 |===
-| n.name | n.age
+| other.name | other.age
 | "Andy" | 36
 2+|Rows: 1
 |===
 
-
-[[filter-on-relationship-type]]
-=== Filter on relationship type
-
-You can put the exact relationship type in the `MATCH` pattern,
-It also possible to do more advanced filtering on the relationship type by using the special property `type` to compare the type with something else.
-In this example, the query does a regular expression comparison with the name of the relationship type:
-
-.Query
-[source, cypher]
-----
-MATCH (n:Person)-[r]->()
-WHERE n.name='Andy' AND type(r) =~ 'K.*'
-RETURN type(r), r.since
-----
-
-This returns all relationships having a type whose name starts with 'K':
-
-.Result
-[role="queryresult",options="header,footer",cols="2*<m"]
-|===
-| type(r) | r.since
-| "KNOWS" | 1999
-| "KNOWS" | 2012
-2+|Rows: 2
-|===
-
-
```

Comment:
This did not belong in this section, as it has nothing to do with path pattern expressions. 

If we need a new section to discuss the `type` function and its siblings `types`, `label`, and `labels`, then I would suggest making a new card for them. There however is already documentation concerning these on the https://neo4j.com/docs/cypher-manual/current/functions/scalar page.